src/pylptlib/lpt.py

```diff
@@ -1367,6 +1373,9 @@ def reset_measurement_dict():
     """Reset the _measurements dictionary for a new set of data."""
     _measurements.clear()
 
+    # Clear the measurement tables of the device
+    clrscn()
+

```

Comment:
Method name and docstring no longer match to what the function does, because of the added clrscn.
I think a better name would be something like `reset_measurement_cache`. But the old function might still be needed for backward compatibility?

Reply:
Good point. As it is only used in one driver at one occasion, I will update it in the driver